SafeguardSessions/Scripts/RunUnitTests.ps1

```diff
@@ -10,17 +7,79 @@ $dir = Split-Path $scriptpath
 
 . $dir\PreCheck.ps1
 
-if ($null -eq $Output) {
-    $Output = "short"
-}
-
-if ($null -eq $TestFile) {
-    $TestFile = $config.unit_tests_path
+$Output = "short"
+if ($null -eq $TestFile){
+    $testInput = $config.unit_tests_path
+} else {
+    $testInput = $TestFile
 }
 
-& pqtest run-test `
+$pqtestOutput =  pqtest run-test `
     --extension $config.connector_mez_path `
     --extension $config.test_mez_path `
     --environmentConfiguration output=$Output `
-    --queryFile $TestFile `
-    --prettyPrint
+    --queryFile $testInput `
+    --prettyPrint 
+
+$resultsAsObject =  $pqtestOutput | ConvertFrom-Json

```

Comment:
#Optional: I'd put these after the function definitions.

Reply:
ACK.